yaqd_wright/_yaqd_wright_filter_wheels_continuous.py

```diff
@@ -0,0 +1,62 @@
+# -*- coding: utf-8 -*-
+"""
+Created on Fri Sep 18 09:53:37 2020
+
+@author: jsche
+"""
+__all__ = ["YaqdWrightFilterWheelsContinuous"]
+
+import asyncio
+import time
+from yaqd_core import ContinuousHardware, aserial
+
+from .__version__ import __branch__
+
+class YaqdWrightFilterWheelsContinuous(ContinuousHardware):
+    _kind = "wright-filter-wheels-continuous"
```

Comment:
I would like to standardize on the `kind` being "wright-continuous-filter-wheel"

This involves:

- [ ] changing `_kind` to `"wright-continuous-filter-wheel"` (MUST match the protocol in the toml file, which is already set to this)
- [ ] changing the class name to `WrightContinuousFilterWheel` (including in `__all__` above)
- [ ] changing the file name to [`_wright_continuous_filter_wheel.py`](https://github.com/wright-group/yaqd-wright/blob/master/yaqd_wright/_wright_continuous_filter_wheel.py)